lib/watir-webdriver/elements/checkbox.rb

```diff
@@ -50,8 +50,5 @@ def checkboxes(*args)
   end # Container
 
   class CheckBoxCollection < InputCollection
```

Comment:
We should change this to one of the "good" approaches, no preference from me on which - https://github.com/bbatsov/ruby-style-guide/blob/master/README.md#single-line-classes